setupGuide.adoc

```diff
@@ -0,0 +1,75 @@
+= Setting Up Your Workspace
+
+:toc: left
+
+== Introduction
+
+This document provides instructions on how to set up your development workspace on both macOS and Windows. Follow the steps in the section that corresponds to your operating system.
+
+== macOS Setup
+
+=== Step 1: Install Golang
+
+Go is a simple and fast programming language, developed in Google.
+
+Download and install Golang from the official website: https://go.dev/dl/
+
+To check installation and version (1.22.0), type:
+
+## [source, terminal]
```

Comment:
pretty sure the syntax is wrong. GitHub can display adoc files. Please check and change.